datafusion/core/src/execution/context.rs

```diff
@@ -1253,12 +1253,14 @@ impl SessionState {
                 .register_catalog(config.default_catalog.clone(), default_catalog);
         }
 
+        let execution_props = ExecutionProps::new();
+
         SessionState {
             session_id,
             optimizers: vec![
                 // Simplify expressions first to maximize the chance
                 // of applying other optimizations
-                Arc::new(SimplifyExpressions::new()),
+                Arc::new(SimplifyExpressions::new(execution_props.clone())),
```

Comment:
This is not correct. We need the same `ExecutionProps` to be shared between `SessionState` and `SimplifyExpressions` so that they both have the same execution start time.